src/components/RadioField.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div>
+        <b-form-group
+            label-cols-sm="4"
+            label-cols-lg="3"
+            :description="field.description"
+            :label="field.name"
+            :label-for="`field-${field.name}`"
+            :invalid-feedback="error.content"
+            valid-feedback="Ce champ est valide"
+            :state="isValid"
+          >
+            <b-form-radio-group v-model="value" :id="`field-${field.name}`" :options="options" v-on:input="onInput" :state="isValid" />
+        </b-form-group>
+    </div>
+</template>
+<script>
+import ValidateField from '@/mixins/ValidateField.vue'
+
+export default {
+    name: 'RadioField',
+    mixins: [ValidateField],
+    data() {
+        let options = [
+            { text: 'Vrai', value: 'true' },
+            { text: 'Faux', value: 'false' },
+        ]
+
+        if (this.containsKey(this.field, "constraints") && this.containsKey(this.field.constraints, "required") && this.field.constraints.required) {
```

Comment:
Is there a shorthand for this?

Reply:
`if (this.field.constraints && this.field.constraints.required)` doesn't work?

If not maybe you can extract the constraints in a list in the parent Class and just do a `indexOf`.